docs/docs/07-cli.md

```diff
@@ -30,6 +30,7 @@ ignite scaffold chain github.com/username/mars
 
 * [ignite account](#ignite-account)	 - Commands for managing accounts
```

Comment:
Can we avoid updating the md files because the docs website updated through the develop branch and we have some unreleased features documented here.

Reply:
sure, done with new commit